sam-mongodb/src/sam_mongodb/lifecycle.py

```diff
@@ -71,7 +71,7 @@ def initialize_mongo_agent(host_component: Any, init_config: MongoAgentInitConfi
 
     try:
         db_service = MongoDatabaseService(connection_params, init_config.query_timeout)
-        log.info(
+        log.debug(
```

Comment:
I would keep the key lifecycle events at INFO. This is the initialization and cleanup (line 167) logs. 

Lifecycles related logs are often important to confirm states of the system. We want to keep them especially when they only run once at startup & shutdown.